test/unit/alignment/configuration/align_config_edit_test.cpp

```diff
@@ -30,7 +32,7 @@ TEST(align_cfg_edit, is_hamming)
 
 TEST(align_cfg_edit, is_simple_gap)
 {
```

Comment:
```suggestion
{
    using seqan3::get;

```

Reply:
Why did you remove `seqan3::get` ?

Reply:
To quote @rrahn :

This is a bug in the implementation of the configuration. So you need to call it unqualified, i.e. `get<seqan3::align_cfg::gap_cost_affine>(cfg)`.
The reason for this is, that the configuration class inherits from `std::tuple` and also it's get interface. But we added one additional interface to call it with template types. This was ok before, since `seqan3::align_cfg::gap` was a class template. But now this changed and `seqan3::align_cfg::gap_cost_affine` is a regular class without templates. In this case it was not finding a suitable candidate, because it had to use `std::get` instead.